android-project/app/build.gradle

```diff
@@ -0,0 +1,101 @@
+plugins {
+    id 'com.android.application'
+}
+def androidProject = projectDir.parentFile
+
+android {
+    namespace "org.legoisland.isle"
+    compileSdk 35
+
+    defaultConfig {
+        applicationId 'org.legoisland.isle'
+        minSdk 21
+        targetSdk 35
+        versionCode 1
+        versionName '1.0'
+
+        externalNativeBuild {
+            cmake {
+                arguments '-DANDROID_STL=c++_shared', "-DFETCHCONTENT_BASE_DIR=${androidProject}/build/_deps"
+                if (project.hasProperty('cmakeArgs')) {
+                    project.cmakeArgs.split(" ").each {arg -> arguments arg}
+                }
+                abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
+            }
+        }
+    }
+    externalNativeBuild {
+        cmake {
+            version "3.30.5"
```

Comment:
Is this line required?
[SDL_helloworld does not have it.](https://github.com/libsdl-org/SDL_helloworld/blob/501e31c4d9deac135b63b7535c2220d7d71d4d92/android-project/app/build.gradle#L25-L29)